psalm.xml

```diff
@@ -15,6 +15,7 @@
         <directory name="tests"/>
         <ignoreFiles>
             <directory name="tests/Application"/>
+            <directory name="tests/Functional"/>
```

Comment:
I try to avoid this, but I guess you have your reasons? :D

Reply:
Psalm is complaining about tons of stuff like nullable getters or too generatic type hints - things that could be important in the application code but that are just  irrelevant in the tests, where we have 150% over data.
Still, probably skipping the whole directory is a little exaggeration 💃 I will try to be more precise 🖖

Reply:
Yeah, I figured it out when I read your tests. It's fine :D